core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.component.html

```diff
@@ -41,6 +42,82 @@
   </tbody>
 </table>
 
+<table *ngIf="isSerialChapterLaunchFeatureEnabled()"
+       class="dashboard-table story-dashboard-table e2e-test-story-list-table">
+  <colgroup>
+    <col class="oppia-colgroup-column-1">
+    <col class="oppia-colgroup-column-2">
+    <col class="oppia-colgroup-column-3">
+    <col class="oppia-colgroup-column-4">
+  </colgroup>
+  <tr>
+    <th *ngFor="let key of STORY_TABLE_COLUMN_HEADINGS"
+        class="story-dashboard-table-headings">
+      <p *ngIf="key === 'title'" class="text-center">Title</p>
+      <p *ngIf="key === 'publication_status'" class="text-center">Status</p>
+      <p *ngIf="key === 'node_count'" class="text-center">Chapters</p>
+      <p *ngIf="key === 'notifications'" class="text-center">Chapter Publication Notifications  </p>
+    </th>
+  </tr>
+  <tbody cdkDropList
+         (cdkDropListDropped)="drop($event)">
+    <tr *ngFor="let summary of storySummaries; index as idx"
+        class="story-list-item e2e-test-list-view-item-item e2e-test-story-list-item"
+        cdkDrag>
+      <td>
+        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary text-left">
+          <span> {{ summary.getTitle() }} </span>
+        </a>
+      </td>
+      <td>
+        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary text-center">
+          <span *ngIf="summary.getPublishedChaptersCount()!=0 && summary.getPublishedChaptersCount()!=summary.getTotalChaptersCount()">
+            Partially Published
+          </span>
+          <span *ngIf="summary.getPublishedChaptersCount() == 0">
+            Not Published
+          </span>
+          <span *ngIf="summary.getPublishedChaptersCount() != 0 && summary.getPublishedChaptersCount() == summary.getTotalChaptersCount()">
+            Fully Published
+          </span>
+        </a>
+      </td>
+      <td>
+        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary chapter-count text-center">
+          <span> {{ summary.getPublishedChaptersCount() }} Published</span>
+          <span *ngIf="areChaptersAwaitingPublication(summary)"> {{ summary.getTotalChaptersCount() - summary.getPublishedChaptersCount() }} Awaiting Publication</span>
+        </a>
+      </td>
+      <td>
+        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary text-left">
+          <span *ngIf="summary.getUpcomingChaptersCount() == 1">
+            {{ summary.getUpcomingChaptersCount() }} upcoming launch (next
+            <span *ngIf="summary.getUpcomingChaptersExpectedDays()[0] > 1">{{ summary.getUpcomingChaptersExpectedDays()[0] }} days</span>
```

Comment:
Put the ngIf branching only around "day/days" or around the "s". In general keep the part that branches as small as possible.

Reply:
Done